core/accounting/src/fiscal_year/repo.rs

```diff
@@ -0,0 +1,28 @@
+use es_entity::*;
+use sqlx::PgPool;
+
+use super::{
+    entity::{FiscalYear, FiscalYearEvent},
+    error::FiscalYearError,
+};
+use crate::primitives::{ChartId, FiscalYearId};
+
+#[derive(EsRepo, Clone)]
+#[es_repo(
+    entity = "FiscalYear",
+    err = "FiscalYearError",
+    columns(
+        chart_id(ty = "ChartId", list_for, parent),
```

Comment:
`parent` is only for entities with nested child entities I believe

```suggestion
        chart_id(ty = "ChartId", list_for),
```